packages/nova-lib/lib/server/meteor_patch.js

```diff
@@ -0,0 +1,98 @@
+import { Meteor } from 'meteor/meteor';
+import { WebApp } from 'meteor/webapp';
+
+import { Utils } from '../modules/index.js';
+
+// our bindEnvironment keep the original function and auto rename the return function
+export const bindEnvironment = (fn) => {
```

Comment:
I understand that having custom `Meteor.bindEnvironment` & `WebApp.connectHandlers.use` brings nice improvements, but do we really need that?

IMO, we should minimize how we patch Meteor 😳